app/packages/core/client/controllers/grits_search.coffee

```diff
@@ -334,7 +352,8 @@ Template.gritsSearch.onRendered ->
     }]
   })
   _setSearchBar(searchBar)
-
+  species = _topSpecies[Math.floor(Math.random() * 14)]
```

Comment:
Maybe we should use the `_topSpecies.length - 1` rather than `14` in case we add or subtract from the array of species?